modules/client-configuration/pages/registration-methods-saltbundle.adoc

```diff
@@ -0,0 +1,30 @@
+[[registration-methods-saltbundle]]
+= Client Registration with Salt Bundle
+
+The registration method with the Salt bundle is the recommended registration method as long as it covers your use case.
+This section explains the advantages and limitations of the current implementation.
+The Salt bundle is provided as the [package]``venv-salt-minion`` that consists of Salt, Python, and the Python modules Salt depends on.
+Using the Salt bundle, in most cases it is no longer needed that the client provides the required Python version or any Python modules.
+Bootstrapping with {webui} still requires Python to be installed on the client.
+
+If you bootstrap new clients, registration with the Salt bundle is the default method.
+You can switch existing clients to the Salt bundle method.
+If you switch, the [package]``salt-minion`` package and its Python version and the Python modules will stay installed.
+
+The Salt state [literal]``util.mgr_switch_to_venv_minion`` is available to switch from [package]``salt-minion`` to [package]``venv-salt-minion``.  It is recommended to switch to [package]``venv-salt-minion`` in two steps:
```

Comment:
I have one question here maybe for @vzhestkov about the switching to the salt bundle.

According to this: `It is recommended to switch to [package]``venv-salt-minion`` in two steps:`

Question: are these steps supposed to be executed sequentially or are they simply two separeted approaches? After reading this, it is not clear to me whether I need to mandatory do the two steps for the switch or it is ok only doing one.

Reply:
These steps should be done sequentially. Actually the second one do the same but in addition remove the config files and the package of original salt minion, but if executed first most possible will fail as can't shift the state apply process to the bundle, so first step is installing the bundle and copying the config, the second stage is cleanup.

Reply:
ok, I tried the the description accordingly. If it is or sounds wrong, please provide review patch.  merging now...